packages/api-docs-builder/ApiBuilders/HookApiBuilder.ts

```diff
@@ -423,6 +428,48 @@ const extractInfoFromType = (typeName: string, project: TypeScriptProject): Pars
   return result;
 };
 
+/**
+ * Helper to get the import options
+ * @param name The name of the hook
+ * @param filename The filename where its defined (to infer the package)
+ * @returns an array of import command
+ */
+const getHookImports = (name: string, filename: string) => {
+  let source;
+
+  if (filename.includes('/packages/mui-material')) {
```

Comment:
Could we just extract the package name from the directory name with something like `/\/packages\/mui-([^/]+)\//`?